console/billing/how-to/change-payment-method.mdx

```diff
@@ -15,22 +15,22 @@ categories:
 
 If you have registered both a **credit card** and a **SEPA mandate**, you can choose the default payment method you wish to use for your Scaleway account.
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
-  - You have added a valid credit card on your account, even if you have set up SEPA as the default payment method.
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- A valid credit card added to your account, even if you have set up SEPA as the default payment method.
+
```

Comment:
I think you might also need IAM permissions to add a billing contact, so we should add the requirement about being the Owner or having appropriate permissions- @ldecarvalho-doc  can you confirm?